### PR TITLE
Camera adding

### PR DESCRIPTION
**Pull request by UralmashFox**
I don't know, what will be if I push "create pull request, but I try"...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,9 @@ rosrun thespot_python main.py
 
 ```
 This magic python code makes my creation walk :3 Just impressive, my baby does it's first steps ^_^
```
Comment (l1va) [inline README.md:30]: Happy birthday! Wish you fewer bugs and green CI

File: spot/src/myrobot/urdf/thespot.urdf.xacro [hunk 2]
```diff
@@ -92,6 +93,11 @@
 
 			<!-- A bunch of stuff cut -->
    </xacro:macro>
+<<<< camera_adding
```
Comment (l1va) [inline spot/src/myrobot/urdf/thespot.urdf.xacro]: missed conflict line

File: spot/src/thespot_python/scripts/main.py [hunk 3]
```diff
@@ -54,13 +56,31 @@ def talker():
...
         pubRBS.publish(positionRBS)
 
-
-
         rate.sleep()
-        angle += 0.9
+        angle += 0.5
+
+""" def callback(data):
+    rospy.loginfo("I heard %s",data.data)
```
Comment (l1va) [inline spot/src/thespot_python/scripts/main.py]: i think this code is not needed here anymore, remove then

File: spot/src/thespot_python/scripts/subscriber.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        print(e)
+    else:
+        # Save your OpenCV2 image as a png
+        #time = str(msg.header.stamp)
+
+        t = str(int(time.time()) - start_time + 1)
+        print(t)
+
+    
+        cv2.imwrite("/home/marina/spot/pic"+t+".png", cv2_img)
```
Comment (l1va) [inline spot/src/thespot_python/scripts/subscriber.py:30]: hardcode, it will not work on my laptop :(

**Review by l1va — CHANGES_REQUESTED** [review]

**Comment by UralmashFox** [comment]
Now camera is added to the model.

**Pull request merged**